index.js

```diff
@@ -6,96 +6,76 @@ var Funnel = require('broccoli-funnel');
 var mergeTrees = require('broccoli-merge-trees');
 var compileSass = require('broccoli-sass-source-maps');
 
+// Fetch a list of known backends. The user can always choose to override any of these URLs via ENV vars
+var knownBackends = require('./config/backends');
```

Comment:
That's a relief; I've been burned by ES6 in node4 before, since `index.js` doesn't appear to get transpiled. Huzzah for quality of life!